litmus-portal/backend/auth/pkg/providers/github/github.go

```diff
@@ -2,16 +2,20 @@ package github
 
 import (
 	"context"
+	"fmt"
+	"log"
 	"net/http"
 
+	"github.com/gin-gonic/gin"
+	"github.com/google/go-github/v32/github"
 	"golang.org/x/oauth2"
 	githubAuth "golang.org/x/oauth2/github"
 )
 
 var (
 	config = oauth2.Config{
-		ClientID:     "3a6b324047494810c849",
-		ClientSecret: "6671f33aa589f324a2491ae418ff2706f997b991",
+		ClientID:     "ef1f3bef5f901dec6c9d",
+		ClientSecret: "2723a84e77bae8602e45259cc07c6af85a9dc3ca",
```

Comment:
Are these actual github api secrets? If they are please remove them immediately and reset the secrets.